prototypeGui.py

```diff
@@ -0,0 +1,393 @@
+"""
+This program creates a GUI for exporting combustion
+experiment data to a YAML file in the ChemKED format.
+"""
+
+import sys
+from PyQt5.QtWidgets import (QApplication, QWidget, QMainWindow,
+        QPushButton, QMessageBox, QAction, QToolTip, QDesktopWidget, QSpinBox,
+        QHBoxLayout, QVBoxLayout, QTabWidget, QSplashScreen,
+        QLabel, QLineEdit, QFormLayout, QGroupBox, QScrollArea)
+from PyQt5.QtGui import *
+
+
+class Window(QMainWindow):
+    """
+    Controls main window handling.
+    """
+    def __init__(self):
+        # Inherits class methods from QMainWindow
+        super(Window, self).__init__()
+
+        self.title = 'Prototype ChemKED GUI'
+        self.left = 0
+        self.top = 0
+        self.width = 800
+        self.height = 600
+        self.setWindowTitle(self.title)
+        self.setGeometry(self.left, self.top, self.width, self.height)
+        self.setWindowIcon(QIcon('pyked-logo.png'))
+
+        self.vbox = QVBoxLayout()
+
+        # The Table Class handles the majority of the work.
+        self.table_widget = Table(self)
+        self.vbox.addWidget(self.table_widget)
+        self.setCentralWidget(self.table_widget)
+
+        self.center()
+
+        # Easily close the GUI (without saving)
+        close_gui = QAction("&Close GUI", self)
+        close_gui.setShortcut('Ctrl+Q')
+        close_gui.setStatusTip('Close without saving.')
+        close_gui.triggered.connect(self.closeEvent)
+
+        self.show()
+
+
+    def center(self):
+        """
+        Centers the window on the screen.
+        """
+
+        # Get window geometry.
+        qr = self.frameGeometry()
+        # Get screen resolution.
+        cp = QDesktopWidget().availableGeometry().center()
+        # Move center of rectangle of equal size to geometry
+        #  to the center of the screen.
+        qr.moveCenter(cp)
+        # Move the top left corner of the geometry to the
+        #  top left corner of the rectangle.
+        self.move(qr.topLeft())
+
+
+    def closeEvent(self, event):
+        """
+        Prompts the user before closing window.
+        """
+        reply = QMessageBox.question(self, 'Message',
+            "Are you sure to quit?", QMessageBox.Yes |
+            QMessageBox.No, QMessageBox.Yes)
+
+        if reply == QMessageBox.Yes:
+            event.accept()
+        else:
+            event.ignore()
+
+
+class Table(QWidget):
+    """
+    This class handles creating tabs, their contents, and exporting to YAML.
+    In each tab is file information.
+    When the Export button is clicked, the information
+    entered is exported to a YAML file.
+    """
+
+    def __init__(self, parent):
+        """
+        Sets up tabs and contents.
+        """
+        super(QWidget, self).__init__(parent)
+        self.layout = QVBoxLayout(self)
+
+        # Initialize tab screen
+        self.tabs = QTabWidget()
+        self.tab1 = QWidget()
```

Comment:
Consider naming these tabs something useful, based on the rough classes of information in the documentation: https://pr-omethe-us.github.io/PyKED/schema-docs.html